build.xml

```diff
@@ -8,119 +8,268 @@
 
 
 <project name="xspec" default="xspec">
-	<description>XSpec is a Behavior Driven Development (BDD) framework for XSLT and XQuery.
-
-Usage:
-
-ant -Dxspec.xml=filename
-
-where:
-
-  filename          XSpec tests XML file
-
-
-</description>
-
+  <description>XSpec is a Behavior Driven Development (BDD) framework for XSLT and XQuery.
+    
+    Usage (command-line):
+    
+    ant -Dxspec.xml=/path/to/xspec/file
+    -Dtest.type=t
+    -Dxspec.compiled.xsl.dir=/path/to/folder 
+    -Dxspec.project.dir=/path/to/folder/containing/buildfile 
+    -Dxspec.phase=#ALL
+    -lib /path/to/saxon.jar 
+    -lib /path/to/xmltask-1.16.jar 
+    -lib /path/to/ant-contrib-1.0b3.jar 
+    
+    where:
+    
+    test.type                  Value "t" if XSpec describes an XSLT file, "s" if XSpec describes a Schematron file
+                               [Optional; the default value is "t"]
+    xspec.phase                Required if XSpec describes a Schematron file, with value "#ALL" 
+    xspec.compiled.xsl.dir     Folder in which to place the compiled XSLT file generated from a Schematron file
+                               [Optional; the default is a subfolder "xspec" of the folder containing the XSpec file]
+    
```

Comment:
Not all parameters are explained. Are they missed? Or, will be explained only in Wiki?

Reply:
I added more documentation in https://github.com/xspec/xspec/commit/e9861f8045f20806b05725e3b0cf53ea1180732e. This will be used to update the wiki. I left the documentation on how to run it in oXygen as it will be useful for the oXygen integration and for all those who want to use it before this feature is fully integrated in oXygen.